adp-algorithms/bell-brockhausen/src/main/java/de/metanome/algorithms/bellbrockhausen/BellBrockhausenAlgorithm.java

```diff
@@ -35,25 +34,33 @@ public BellBrockhausenAlgorithm() {
     public ArrayList<ConfigurationRequirement<?>> getConfigurationRequirements() {
         final ArrayList<ConfigurationRequirement<?>> requirements = new ArrayList<>();
         requirements.add(new ConfigurationRequirementDatabaseConnection(DATABASE.name()));
-        requirements.add(new ConfigurationRequirementString(TABLE.name()));
+        requirements.add(new ConfigurationRequirementTableInput(TABLE.name()));
         return requirements;
     }
 
-    @Override
+  @Override
+  public void setTableInputConfigurationValue(final String identifier, final TableInputGenerator... values)
+  throws AlgorithmConfigurationException {
+
+    if (identifier.equals(TABLE.name())) {
```

Comment:
Have you tested this? Where do you get your database connection when using the `TABLE` identifier?